ext/curl/curl_private.h

```diff
@@ -125,11 +136,13 @@ typedef struct {
 } php_curlm;
 
 typedef struct _php_curlsh {
-	CURLSH                   *share;
+	CURLSH *share;
+
 	struct {
 		int no;
 	} err;
-	zend_object std;
+
+    zend_object std;
 } php_curlsh;
```

Comment:
Unrelated changes: Prefer to keep the diff minimal.

Reply:
That's fair, especially since I somehow messed up the spacing on the `zend_object` line - my Emacs setup is not tabs-friendly, haha. I do think that whitespace makes these and other structs more readable, but I'll remove this.